src/libNode/Node.cpp

```diff
@@ -1918,3 +1912,27 @@ std::string Node::GetActionString(Action action) const {
   }
   return true;
 }
+
+void Node::GetEntireNetworkPeerInfo(std::vector<std::pair<PubKey, Peer>>& peers,
+                                    std::vector<PubKey>& pubKeys) {
+  peers.clear();
+  pubKeys.clear();
+
+  for (const auto& i : *m_myShardMembers) {
+    if (i.second.m_listenPortHost != 0) {
+      peers.emplace_back(i);
+    }
```

Comment:
Can add `continue` and skip adding my own pubkey?

Reply:
Fixed. Adding `continue` is not relevant though.